pkg/quarterdeck/orgs.go

```diff
@@ -159,21 +160,21 @@ func (s *Server) ProjectCreate(c *gin.Context) {
 	// Bind the Project request to the project data structure
 	if err = c.BindJSON(&project); err != nil {
 		sentry.Warn(c).Err(err).Msg("could not parse project create request")
-		c.JSON(http.StatusBadRequest, api.ErrorResponse(api.ErrUnparsable))
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(responses.ErrTryProjectAgain))
 		return
 	}
 
 	// Validate the request from the API side.
 	if err = project.Validate(); err != nil {
 		c.Error(err)
-		c.JSON(http.StatusBadRequest, api.ErrorResponse(err))
+		c.JSON(http.StatusBadRequest, api.ErrorResponse(responses.ErrTryProjectAgain))
 		return
 	}
 
 	// Fetch the user claims from the request
 	if claims, err = middleware.GetClaims(c); err != nil {
 		sentry.Error(c).Err(err).Msg("could not get user claims from authenticated request")
-		c.JSON(http.StatusUnauthorized, api.ErrorResponse(api.ErrInvalidUserClaims))
+		c.JSON(http.StatusUnauthorized, api.ErrorResponse(responses.ErrNeedPermission))
```

Comment:
Would `ErrSomethingWentWrong` be a little more appropriate here since an error is returned if the user's claims aren't fetched?

Reply:
Does that mean the error should be `http.StatusInternalServerError` instead of `http.StatusUnauthorized`?

Reply:
I took a look at the `GetClaims` method and it returns an error if no claims exist on the context, so I feel that's why `http.StatusUnauthorized` is used instead of `http.StatusInternalServerError`. In that case, informing the user they need permission seems fine, but should they be directed to their owner in this scenario?